### DISCUSSION TITLE
Error in generating  Bin2AsciiLevel1.e gps1x2rnx.e from make

### DISCUSSION THREAD

**Discussion by chethanram7009** [jgte/grace-data#5] [Q&A] [answered]
Hello all,
I was following all the instructions given in README file to convert level1B data from .dat to .asc format. I had checked for the installation of make, it is available. But when I tried to run the "make" command, there is no any .Bin2AsciiLevel1.e gps1x2rnx.e files are generated.  I had attached the screenshot of the error, it'll be really helpful if someone can help me to solve this error.

![Error_make](https://github.com/user-attachments/assets/6fd6afbe-7865-4067-aeef-17899ac42935)

**Comment by jgte** [comment, marked as answer]
Hello,

I've seen those or similar errors messages when compiling old code with newer compilers. I suggest you try compiling with older compilers.

As a side note, it is preferable to copy-paste the errors messages because that makes them findable to other people.

**Reply by chethanram7009** [reply]
Thank you. It got solved by using lower version compilers.
